docs/couchdb.md

```diff
@@ -0,0 +1,49 @@
+# `couchdb` Metrics Receiver
+
+The couchdb receiver can retrieve stats from your couchdb server using the `/_node/_local/_stats/couchdb` endpoint.
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your Apache web server configuration.
+
+To configure a receiver for your Apache web server metrics, specify the following fields:
+
+| Field                 | Default                 | Description                                                                            |
+|-----------------------|-------------------------|----------------------------------------------------------------------------------------|
+| `type`                | required                | Must be `couchdb`.                                                                     |
+| `endpoint`            | `http://localhost:5984` | The url exposed by couchdb                                                             |
+| `username`            | not set by default      | The username used to connect to the server.                                            |
+| `password`            | not set by default      | The password used to connect to the server.                                            |
+| `collection_interval` | `60s`                   | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    couchdb:
+      type: couchdb
+      endpoint: http://localhost:5984
+      username: otelu
+      password: otelp
+  service:
+    pipelines:
+      couchdb:
+        receivers:
+          - couchdb
+```
+
+## Metrics
+
+The Ops Agent collects the following metrics from your couchdb server.
+
+| Metric                                               | Data Type | Unit       | Labels                      | Description                                  |
+|------------------------------------------------------|-----------|------------|-----------------------------|----------------------------------------------|
+| workload.googleapis.com/couchdb.average_request_time | gauge     | ms         | node_name                   | The average duration of a served request.    |
+| workload.googleapis.com/couchdb.httpd.bulk_requests  | sum       | requests   | node_name                   | The number of bulk requests.                 |
```

Comment:
These custom units are all `{requests}`, `{views}`, etc., right? (with the curly brackets)

Reply:
fixed